### DISCUSSION TITLE
Please develop a way to check if a queue is forming in the Sony Playstation site.

### DISCUSSION THREAD

**Discussion by rjamor** [jef/streetmerchant#1049] [Help]
### Please develop a way to check if a queue is forming in the Sony Playstation site.

There was a queue forming in the Sony Playstation site for the PS5 consoles, but your checker did not detect any stock.

**Comment by willwhitney** [comment]
A simple way to do this would be to check whether we get redirected to direct-queue.playstation.com, but I'm not sure how to do this in the current Store model.
